docs_user/assemblies/proc_adopting-the-loadbalancer-dataplane.adoc

```diff
@@ -0,0 +1,41 @@
+[id="adopting-the-loadbalancer-dataplane_{context}"]
```

Comment:
Is this assembly not used from anywhere yet?

Reply:
you are right. Fixed.